Formula/p/postgresql-latest.rb

```diff
@@ -0,0 +1,76 @@
+class PostgresqlLatest < Formula
+  desc "Meta formula to install the latest PostgreSQL"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v17.0/postgresql-17.0.tar.bz2", using: :nounzip
```

Comment:
Given this isn't used, can probably reduce download size by using one of the checksum files, e.g.
```suggestion
  url "https://ftp.postgresql.org/pub/source/v17.0/postgresql-17.0.tar.bz2.md5"
```